runtime/src/lib.rs

```diff
@@ -291,8 +322,17 @@ parameter_types! {
 	pub const MaxProposals: u32 = 100;
 	pub const PreimageByteDeposit: Balance = GLMR / 1_000;
 	pub const InstantAllowed: bool = false;
+
+	pub const Ensure100: u32 = 100;
 }
 
+type EnsureUnanimousCouncil = pallet_collective::EnsureProportionAtLeast<
+	Ensure100,
+	Ensure100,
+	AccountId,
+	pallet_collective::Instance1,
+>;
```

Comment:
This is a bit hard to read. What do you think about a PR to substrate that adds this to pallet_collective:

```Rust
pub struct EnsureUnanimous<AccountId, I=DefaultInstance>(
	sp_std::marker::PhantomData<(AccountId, I)>
);

impl <O, AccountId, I> EnsureOrigin<O> for EnsureUnanimous<AccountId, I> {
  // --snip--
}
```

Reply:
Oh actually, you did exactly that with a type alias instead of a reimplementation. I guess the one ugly part is the `pub const Ensure100: u32 = 100;` Like any number would have worked there.

Reply:
Yeah it's ugly but you can't directly put u32 in there, only an impl of `Get<u32>`